tests/typ/visualize/gradient-conic.typ

```diff
@@ -0,0 +1,25 @@
+// Test conic gradients
+
+---
+#square(
+  size: 50pt,
+  fill: gradient.conic(..color.map.rainbow, space: color.hsv),
+)
+
+---
+#square(
+  size: 50pt,
+  fill: gradient.conic(..color.map.rainbow, space: color.hsv, center: (10%, 10%)),
+)
+
+---
+#square(
+  size: 50pt,
+  fill: gradient.conic(..color.map.rainbow, space: color.hsv, center: (90%, 90%)),
+)
+
+---
+#square(
+  size: 50pt,
+  fill: gradient.conic(..color.map.rainbow, space: color.hsv, angle: 90deg),
+)
```

Comment:
It's not that big of a deal, but maybe you could reconfigure your editor to just insert the final newline. The `.editorconfig` already prescribes this, so I'm not sure why it's not doing it.

Reply:
Ah yeah, I have the extension "EditorConfig for VS Code" installed. This should really be built-in ...